site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2778,6 +2778,8 @@ public function mergeThread($parent_thread_id, $child_thread_id, &$message, &$ch
                 return false;
             }
 
+            $child_thread_title = "Merged Thread Title: ".$child_thread_title;
```

Comment:
Scanning through the list again, can we also add a `\n` to the end of this string to create a cleaner visual distinction? Right now, a merged thread will look like:
```
Merged Thread Title: E2E Test 1
This is some content that blah blah blah
```

where I think adding a line will (in addition to the prefix string) read things cleaner/easier:
```
Merged Thread Title: E2E Test 1

This is some content that blah blah blah
```